decko/lib/decko/tasks/decko.rake

```diff
@@ -1,6 +1,6 @@
 require "decko/application"
 require_relative "alias"
-require_relative "../../../../card/db/seed_consts"
```

Comment:
card isn't just "card" when installed as a normal gem.

I moved seed_consts into lib as the most obvious way to make it available. Open to better ideas.